lib/fathead/sass/parse_functions.py

```diff
@@ -229,18 +262,19 @@ def create_file(self):
                     method_signature = data_element.get('method_signature').encode('utf-8')
                     description = data_element.get('description').encode('utf-8')
                     name = data_element.get('function').encode('utf-8')
+
                     abstract = str(method_signature) + '<br>' + str(description)
                     url = data_element.get('url').encode('utf-8')
                     list_of_data = [
                         name,                       # unique name
                         'A',                        # type is article
                         '',                         # no redirect data
                         '',                         # ignore
-                        '',                         # no categories
+                        'functions',                # no categories
```

Comment:
I made a mistake, this needs to be more specific: `Sass Functions`. It should only be added if the current article is actually a function though